string message

int16 v
int16 w
int32 send_CAN_counter
int32 get_CAN_counter

int32[] set_motor_rpm
int32[] set_motor_Nm
float64[] motor_rpm
float64[] motor_Nm
int64[] motor_break_code
int16[] motor_tempoC
int16[] mdriver_tempoC
int32[] motor_odom_er






